biggraphite/drivers/cassandra.py

```diff
@@ -95,6 +95,12 @@
     "enable_metrics": bool,
     "writer": lambda k: None if k is None else int(k),
     "replica": lambda k: 0 if k is None else int(k),
+    "meta_write_consistency": str,
```

Comment:
maybe add a proper validator here ?
The validator would take either a string, that it would convert to a ConsistencyLevel, or directly a consistency level (see https://github.com/datastax/python-driver/blob/5c5a53ef8de50a1da48964e9bc8e196f3780ec4b/cassandra/__init__.py#L95)